src/components/MyObservations/ToolbarContainer.js

```diff
@@ -122,13 +122,15 @@ const ToolbarContainer = ( {
       return t( "Uploading-x-of-y-observations", translationParams );
     }
 
+    if ( numUnuploadedObs && numUnuploadedObs !== 0 ) {
+      return t( "Upload-x-observations", { count: numUnuploadedObs } );
+    }
+
```

Comment:
This might be a bit off spec, but if you upload one of several obs, I think it should say there are still obs to upload when it's done.

Reply:
When this branched off the "Upload success" message was sticking around forever, current main is it will disappear after 5s and then the "Upload x observations" state will reappear if there are `numUnuploadedObs !== 0 `. The way I handled it in my PR is to reset the upload state and therefore set `uploadsComplete` to false after 5s. With your changes, it will switch back to "Upload x observations" immediately. I am fine with merging this in if it seems more appropriate UX.

Reply:
I like your approach better, and I think that's what Abhas wants. I merged from main and tried to get the single-upload case to have that 5s delay as well.